salt/_logging/impl.py

```diff
@@ -290,7 +290,7 @@ def _log(self, level, msg, args, exc_info=None,
         else:
             LOGGING_LOGGER_CLASS._log(
                 self, level, msg, args, exc_info=exc_info, extra=extra,
-                stack_info=stack_info, stack_level=stack_level
+                stack_info=stack_info, stacklevel=stack_level
```

Comment:
this fixes https://jenkinsci.saltstack.com/job/Full-PR/job/full-pr-arch-lts/job/PR-1622/3/console vs https://jenkinsci.saltstack.com/job/Full-PR/job/full-pr-arch-lts/job/PR-1622/4/console

Reply:
Yep, I had this fix on another PR. Approved.